base/operators.jl

```diff
@@ -189,7 +189,7 @@ copy(x::Union{Symbol,Number,AbstractString,Function,Tuple,LambdaStaticData,
               TopNode,QuoteNode,DataType,Union}) = x
 
 # function pipelining
-|>(x, f::Callable) = f(x)
+|>(x, f) = f(x)
```

Comment:
Won't this conflict with the deprecated piping of `Cmd` objects via `|>` ? I don't think we can make this change until after that deprecation gets removed.